src/components/markdown/component-mappings/index.ts

```diff
@@ -0,0 +1,14 @@
+import P from './p';
```

Comment:
@mcolella14 not completely opposed to this approach but was curious what you thought about having a `index.js` file in a folder called something like `typography` or `textElements` and then have each of the HTML text elements exported so you could do the below to import them:

`import { H2, A, P } from './textElements';
`
Suggestions only because each file doesn't have much going on, and each item is somewhat related to HTML text elements. But not entirely against your approach was just curious.

Reply:
Yeah I see your point, but `A` and `Youtube` will get a bit bigger once Justin fully implements them so I kinda wanted to establish a pattern of separation, but yeah it is annoying to import separately and there's a lot of files with not much going on. I guess I'll leave it up to @justinmckenzie since he'll be implementing these.

Reply:
yeah you can leave for now and we can always refactor later once we get a better idea of what it will look like. personally I like items extracted anyway so fine with this for now